apps/console/src/features/identity-providers/components/settings/attribute-management/attributes-mapping-list.tsx

```diff
@@ -0,0 +1,148 @@
+import { TestableComponentInterface } from "@wso2is/core/models";
+import {
+    AnimatedAvatar,
+    AppAvatar,
+    Code,
+    DataTable,
+    TableActionsInterface,
+    TableColumnInterface
+} from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, ReactNode, useState } from "react";
+import { Header, SemanticICONS } from "semantic-ui-react";
+import { AttributeMappingListItem } from "./attribute-mapping-list-item";
+import { IdentityProviderClaimInterface, IdentityProviderCommonClaimMappingInterface } from "../../../models";
+
+export interface AttributeMappingListProps extends TestableComponentInterface {
+    /**
+     * This contains the mapped attributes list.
+     */
+    attributeMappingsListToShow: Array<IdentityProviderCommonClaimMappingInterface>;
+    /**
+     * A list of claims that is available for selecting. This list
+     * mostly read only, used for dropdowns and such.
+     */
+    availableAttributesList: Array<IdentityProviderClaimInterface>;
+    noDataPlaceholder?: ReactNode;
+    onMappingDeleted: (mapping: IdentityProviderCommonClaimMappingInterface) => void;
+    onMappingEdited: (
+        oldMapping: IdentityProviderCommonClaimMappingInterface,
+        mapping: IdentityProviderCommonClaimMappingInterface
+    ) => void;
+}
+
+export const AttributeMappingList: FunctionComponent<AttributeMappingListProps> = (
+    props: AttributeMappingListProps
+): ReactElement => {
+
+    const {
+        availableAttributesList,
+        attributeMappingsListToShow,
+        onMappingDeleted,
+        onMappingEdited,
+        noDataPlaceholder
+    } = props;
+
+    const [ editingMappings, setEditingMappings ] = useState<string[]>([]);
+
+    const _createTableActions = (): TableActionsInterface[] => {
```

Comment:
`_` is not a standard practice. Please fix in other places as well.

Reply:
Addressed and renamed signatures as necessary. b49a285